db.py

```diff
@@ -4,11 +4,57 @@
 
 parse.uses_netloc.append("postgres")
 url = parse.urlparse(os.environ["DATABASE_URL"])
+#WILL CHANGE THIS TABLE LATER TO THE ONE WE ARE ACTUALLY USING
+table = "test_news_table"
 
 conn = psycopg2.connect(
```

Comment:
Might want to add try-except block just in case
```
try:
    conn = psycopg2.connect(
        database=url.path[1:],
        user=url.username,
        password=url.password,
        host=url.hostname,
        port=url.port
    )
except:
    print("Cannot connect to database:")
```